Plugin Cubot/src/main/java/net/simon987/cubotplugin/CubotInventory.java

```diff
@@ -38,7 +38,11 @@ public void handleInterrupt(Status status) {
             getCpu().getRegisterSet().getRegister("B").setValue(cubot.getHeldItem());
 
         } else if (a == INV_CLEAR) {
-            if (cubot.spendEnergy(100)) {
+            if (cubot.getHeldItem() == 0x0001) {
+                cubot.storeEnergy(4000);
```

Comment:
I'd prefer that the energy amount would be fetched from config.properties

Reply:
Good idea, more flexible that way.